lib/atoms/components/text-area/index.tsx

```diff
@@ -1,14 +1,20 @@
 export const Textarea = ({
   className,
+  placeholder,
   type,
+  value,
   icon,
   iconPosition,
+  onChange,
   ...props
 }: {
   className?: string;
+  placeholder?: string;
+  value?: string;
   type?: string;
   icon?: any;
   iconPosition?: string;
+  onChange: (e: any) => void;
```

Comment:
```suggestion
  onChange: (e: string) => void;
```
Why we are not using "string" type?

Reply:
![Screenshot (21)](https://github.com/Bcrumbs-net/showcase/assets/106822042/acd27f18-8681-4265-aaaf-9cc799fe6dab)

For fix this error

Reply:
done